### DISCUSSION TITLE
请问大神请问下华硕AC86U的官改固件386要安装哪个版本的aliyundrive-webdav呢？

### DISCUSSION THREAD

**Discussion by mermanho** [messense/aliyundrive-webdav#571] [Q&A]
请问大神请问下华硕AC86U的官改固件386要安装哪个版本的aliyundrive-webdav呢？380和384都无法安装了，谢谢

【2022年08月23日 19:57:15】: ========================== step 1 ===============================
【2022年08月23日 19:57:15】: 开启插件离线安装！
【2022年08月23日 19:57:15】: 检测到你上传的离线安装包：aliyundrivewebdav-merlin-arm384-armv7.tar.gz，安装包大小: 2.0M
【2022年08月23日 19:57:15】: 安装包md5sum校验值：75fc007e557b020d64d5f24d4e25dc62
【2022年08月23日 19:57:15】: 尝试解压离线安装包离线安装包...
【2022年08月23日 19:57:16】: 安装包解压成功！继续！
【2022年08月23日 19:57:16】: 当前jffs分区剩余：31544KB, 空间满足，继续安装！
【2022年08月23日 19:57:16】: 准备安装插件：aliyundrivewebdav
【2022年08月23日 19:57:16】: 运行安装脚本...
【2022年08月23日 19:57:16】: ========================== step 2 ===============================
【2022年08月23日 19:57:16】: 本插件适用于【koolshare merlin aarch64 / merlin_hnd】固件平台，你的平台不能安装！！！
【2022年08月23日 19:57:16】: ========================== step 3 ===============================
【2022年08月23日 19:57:16】: ks_tar_install.sh: 因为插件aliyundrivewebdav安装失败！退出离线安装！
【2022年08月23日 19:57:16】: 删除文件夹-1：/tmp/aliyundrivewebdav
【2022年08月23日 19:57:17】: 删除文件：/tmp/aliyundrivewebdav-merlin-arm384-armv7.tar.gz
【2022年08月23日 19:57:17】: ============================ end ================================

**Comment by master-yun-yun** [comment]
本插件适用于【**### koolshare merlin aarch64 / merlin_hnd**】固件平台，你的平台不能安装！！！

**Comment by crazygit** [comment]
我用[aliyundrivewebdav-merlin-arm380-armv7.tar.gz](https://github.com/messense/aliyundrive-webdav/releases/download/v1.10.1/aliyundrivewebdav-merlin-arm380-armv7.tar.gz)是可以的

我路由器也是华硕AC86U,刷的是固件是梅林改版固件`386.3_2`
